samples/echo/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        applicationId 'com.google.sample.oboe.echo'
+        minSdkVersion 24
+        targetSdkVersion 27
+        versionCode 1
+        versionName '1.0'
+        ndk {
+            abiFilters 'armeabi-v7a', 'arm64-v8a'
+        }
+        externalNativeBuild {
+            cmake {
+                arguments '-DANDROID_STL=c++_shared', '-DANDROID_TOOLCHAIN=clang',
```

Comment:
Why are these needed?

Reply:
Here's my build.gradle for MegaDrone: https://github.com/google/oboe/blob/master/samples/MegaDrone/build.gradle

Reply:
the default is NOT libc, it is clang.  we should make new samples to use the latest one.

unfortunately on api-16, there is no libc, so application compiled on libc++ will not be able to run. default gnustl is still needs to be used